modules/oc-mirror-operator-catalog-filtering.adoc

```diff
@@ -166,7 +163,7 @@ mirror:
           - name: stable
             minVersion: 5.6.0
 ----
-|Within the selected channel of that package, all versions starting with the `minVersion` up to the channel head. This scenario does not relyon the shortest path from the upgrade graph.
+|Within the selected channel of that package, all versions starting with the `minVersion` up to the channel head. The `defaultChannel` should be used in case the filtered channels are not the default.
```

Comment:
```suggestion
|Within the selected channel of that package, all versions starting with the `minVersion` up to the channel head. You must use the `defaultChannel` field in case the filtered channels are not the default.
```

Same comment